doc/api/http2.md

```diff
@@ -2598,3 +2598,16 @@ if the stream is closed.
 [Readable Stream]: stream.html#stream_class_stream_readable
 [`ServerRequest`]: #http2_class_server_request
 [`stream.pushStream()`]: #http2_stream-pushstream
+[`http2.Server`]: #http2_class_http2server
+[`http2.SecureServer`]: #http2_class_http2secureserver
+[`request.socket.getPeerCertificate()`]: tls.html#tls_tlssocket_getpeercertificate_detailed
+[Writable Stream]: stream.html#stream_writable_streams
+[`EventEmitter`]: events.html#events_class_eventemitter
+[`response.end()`]: #http2_response_end_data_encoding_callback
+[`response.socket`]: #http2_response_socket
+[`TypeError`]: errors.html#errors_class_typeerror
+[`response.write(data, encoding)`]: http.html#http_response_write_chunk_encoding_callback
+[`response.write()`]: #http2_response_write_chunk_encoding_callback
+[`response.setHeader()`]: #http2_response_setheader_name_value
+[`response.writeHead()`]: #http2_response_writehead_statuscode_statusmessage_headers
+
```

Comment:
Nit: could you remove the extra line at the end (try to keep a single `\n` at the end of the file)